server/apiHandlers.go

```diff
@@ -18,6 +18,7 @@ package main
 
 import (
 	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes"
```

Comment:
As above, I think you need to add the appropriate lib to the BUILD file (based on Travis errors).